samples/nrf_desktop/src/events/Kconfig

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Event options"
+
+comment "Initially logged events"
+
+config DESKTOP_INIT_LOG_BATTERY_STATE_EVENT
+	bool "battery state event"
+	default y
+
+config DESKTOP_INIT_LOG_BATTERY_LEVEL_EVENT
+	bool "battery level event"
+	default y
+
+config DESKTOP_INIT_LOG_BLE_PEER_EVENT
+	bool "ble peer event"
+	default y
+
+config DESKTOP_INIT_LOG_BLE_DISC_COMPLETE_EVENT
+	bool "ble discovery complete event"
+	default y
+
+config DESKTOP_INIT_LOG_BUTTON_EVENT
+	bool "button event"
+	default y
+
+config DESKTOP_INIT_LOG_CONFIG_EVENT
+	bool "config event"
+	default y
+
+config DESKTOP_INIT_LOG_HID_KEYBOARD_EVENT
+	bool "hid keyboard event"
+	default y
+
+config DESKTOP_INIT_LOG_HID_MOUSE_EVENT
+	bool "hid mouse event"
+	default n
```

Comment:
make it default to y and disable in the configuration

Reply:
ok